site/en/docs/extensions/reference/storage/index.md

```diff
@@ -34,70 +74,77 @@ API. For example:
 }
 ```
 
-## Usage
+## Usage {: #usage }
 
-To store user data for your extension, you can use either `storage.sync`:
+The following samples demonstrate the `local`, `sync`, and
+`session` storage areas:
 
-```js
-chrome.storage.sync.set({key: value}, function() {
-  console.log('Value is set to ' + value);
-});
+<web-tabs>
```

Comment:
What is this? Examples should avoid using things that require explanation, but are unrelated to the thing being demonstrated.

Reply:
I'm confused, can you elaborate a bit more? 🤔

Reply:
What is `<web-tabs>`?

Reply:
It's a component, https://developer.chrome.com/docs/handbook/components/#tabs

<img width="750" alt="Screen Shot 2022-11-28 at 11 13 33" src="https://user-images.githubusercontent.com/37001393/204339919-4f7bdf7c-3d92-4b0f-a050-0fc00e95b42a.png">

[Usage - preview](https://deploy-preview-3961--developer-chrome-com.netlify.app/docs/extensions/reference/storage#usage)